openmetadata-ui/src/main/resources/ui/src/components/common/LeftPanelCard/LeftPanelCard.tsx

```diff
@@ -13,18 +13,13 @@
 
 import { Card } from 'antd';
 import { lowerCase } from 'lodash';
-import React, { HTMLAttributes } from 'react';
+import React from 'react';
+import { LeftPanelCardProps } from './LeftPanelCard.interface';
 
-interface CardProps extends HTMLAttributes<HTMLDivElement> {
-  children: React.ReactElement | string;
-  id: string;
-  classes?: string;
-}
-
-const LeftPanelCard = ({ children, id, classes }: CardProps) => {
+const LeftPanelCard = ({ children, id, className }: LeftPanelCardProps) => {
   return (
     <Card
-      className={`${classes} left-panel-card tw-h-full page-layout-v1-left-panel page-layout-v1-vertical-scroll`}
+      className={`${className} left-panel-card tw-h-full page-layout-v1-left-panel page-layout-v1-vertical-scroll`}
```

Comment:
classNames function will be the right choice here! WDYT?

Reply:
Yes.